src/detect-engine-alert.c

```diff
@@ -208,6 +207,8 @@ static void PacketApplySignatureActions(Packet *p, const Signature *s, const Pac
             // nothing to set in the packet
         } else if (pa->action & (ACTION_ALERT | ACTION_CONFIG)) {
             // nothing to set in the packet
+        } else if (pa->action & (ACTION_ACCEPT)) {
+            // DEBUG_VALIDATE_BUG_ON((p->action & ACTION_ACCEPT) == 0); // should be unreachable
```

Comment:
app accept doesn't set an explicit accept, but it might need to for cleaner logic